docs/concepts.md

```diff
@@ -18,24 +18,19 @@ Its interface should be familiar to users of MySQL, Postgres, Oracle, Hive, Pres
 
 The KSQL CLI acts as a client to the KSQL server (see next section).
 
->An alternative to the KSQL CLI is to use the KSQL Experimental UI.
+>An alternative to the KSQL CLI when doing local development and testing is to use the KSQL Experimental UI, see below.
 
-### KSQL Experimental UI
-The KSQL Experimental UI is a web interface for KSQL. It runs on every KSQL Server unless it is disabled. To view the status and avialability of the page visit **http://ksqlserver:8080**
-The results will appear as follows:
->{"KsqlServerInfo":{"version":"0.4","uiEnabled":true,"uiAddress":"http://localhost:8080/index.html"}}
-
-Access the web interface using http://server:8080/index.html 
-
-
-To disable the interface, modify the **ksqlserver.properties** file as below. If the entry is missing, then it will need to be added.
-> ui.enabled=false
 
 #### KSQL Server
 The KSQL server runs the engine that executes KSQL queries, which includes the data processing as well as reading
 data from and writing data to the target Kafka cluster.
 
- 
+#### KSQL Experimental UI
+KSQL currently ships with an experimental web UI, whose goal it is to provide a simple visual wrapper for the interactive KSQL CLI. The experimental UI is meant to be used for local development, testing, and demoing KSQL. If you want to use KSQL interactively in production environments, we recommend the use of the KSQL CLI instead.
+
+* By default, the KSQL Experimental UI runs on every KSQL Server and is accessible at **http://ksqlserver:8080/index.html**.
```

Comment:
Question for clarification: After the latest changes, do I still need to go to http://ksqlserver:8080/index.html or is http://ksqlserver:8080/ sufficient to access the UI?